Mathlib/NumberTheory/NumberField/InfinitePlace/Ramification.lean

```diff
@@ -324,6 +324,19 @@ lemma not_isUnramified_iff_card_stabilizer_eq_two [IsGalois k K] :
   rw [isUnramified_iff_card_stabilizer_eq_one]
   obtain (e|e) := nat_card_stabilizer_eq_one_or_two k w <;> rw [e] <;> decide
 
+lemma isRamified_iff_card_stabilizer_eq_two [IsGalois k K] :
```

Comment:
Does this deserve to be a `simp` lemma?

Reply:
Well, I am not sure the RHS is that much simpler than the LHS...

Reply:
You made a good point so I added the `simp` tag.

Reply:
Mmm, in my opinion it's not a good `simp` lemma: most of the results about ramified extensions will be stated using `IsRamified`, so `simp` would make it harder to use them. It seems unlikely that `simp` can simplify further `Nat.card (Stab w) = 2` to something trivial.

Reply:
If you have in mind specific examples where `Nat.card (Stab w) = 2` becomes something very easy we can state the specific result, and tagging that with `simp`, but honestly if I see `IsUnramified w` I know what's happening, while `Nat.card (Stab w) = 2` seems a more obscure condition.

Reply:
I don't quite recall the discussion, but I'm inclined to agree with Riccardo here. It depends on how `IsRamified` is used in practice (which I'm least qualified to judge), but my guess is that it's not desirable with `IsRamified` to be simp-ed away in a general case...